HAL/keymaster/4.1/JavacardKeymaster4Device.cpp

```diff
@@ -453,9 +598,33 @@ Return<void> JavacardKeymaster4Device::getHmacSharingParameters(getHmacSharingPa
     }
     _hidl_cb(errorCode, hmacSharingParameters);
     return Void();
+#endif
 }
 
 Return<void> JavacardKeymaster4Device::computeSharedHmac(const hidl_vec<HmacSharingParameters>& params, computeSharedHmac_cb _hidl_cb) {
+    /* TODO temporary fix: vold daemon calls performHmacKeyAgreement. At that time when vold calls this API there is no
+     * network connectivity and socket cannot be connected. So as a hack we are calling softkeymaster to
```

Comment:
For below APIs
1. getHardwareInfo 2. getHmacSharingParameters 3. computeSharedHmac. do a proper fix.

Reply:
Done.